iosHyperskillApp/iosHyperskillApp/Sources/Modules/Home/Views/SolveUnlimitedBadgeView.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+
+extension SolveUnlimitedBadgeView {
+    struct Appearance {
+        let solveUnlimitedHorizontalPadding: CGFloat = 8
+        let solveUnlimitedVerticalPadding: CGFloat = 4
+        let solveUnlimitedCornerRadius: CGFloat = 4
+    }
+}
+
+struct SolveUnlimitedBadgeView: View {
+    private(set) var appearance = Appearance()
+
+    var body: some View {
+        Text(Strings.Home.solveUnlimited)
```

Comment:
The text must be configurable because it could be:
1. Solve unlimited
2. Repeat unlimited

Currently incorrect text for the `TopicsRepetitionsCardView`

Reply:
Also, this view should be renamed to something like this `HomeFreemiumBadgeView`